crates/sui-graphql-rpc/Cargo.toml

```diff
@@ -14,11 +14,15 @@ async-trait.workspace = true
 axum.workspace = true
 chrono.workspace = true
 clap.workspace = true
+env_logger = { version = "0.10.0" }
```

Comment:
We use tracing infra and not `log` family of logging libraries so this probably isnt' needed?